style.css

```diff
@@ -319,40 +350,41 @@ a img:active {
   justify-content: space-between;
   padding: 0 20% 0 20px;
   margin-top: 0;
+  width: 250px;
 }
 
 .about {
   background: #1c1a19;
   color: #fff;
   font-family: 'Inter', sans-serif;
+  padding: 0 10% 0 11%;
 }
 
 #t_about {
   margin-top: 120px;
   margin-bottom: 0;
-  width: 327px;
+  max-width: 100%;
   height: 79px;
```

Comment:
:red_circle: Kindly notice that the about me title is not displaying properly. As requested by the previous reviewer, please make sure you fix this design issue. I think you can remove this line `367` in your `style.css` to resolve this problem. How do you think? :hugs:  
You can see, from this short gif, how the height property is impacting your design :arrow_down: 
![Peek 2022-11-05 02-25](https://user-images.githubusercontent.com/26736582/200092530-dcc5caa9-d451-4171-a920-5e697a2c9747.gif)